sdk/attestation/attestation/test/public/attestationTests.spec.ts

```diff
@@ -197,11 +197,35 @@ describe("[AAD] Attestation Client", function() {
     const binaryRuntimeData = base64url.decodeString(_runtimeData);
     const client = createRecordedClient(endpointType);
 
+    {
+      try {
+        await client.attestOpenEnclave(base64url.decodeString(_openEnclaveReport), {
+          runTimeData: binaryRuntimeData,
+          runTimeJson: binaryRuntimeData
+        });
+        assert.fail("Expected attest call to throw.");
+      } catch (TypeError) {
+        assert.isTrue(true);
```

Comment:
not sure what is the point here.

Reply:
This is a variant of assert.throws that works for async functions (I wrote this before I learned of Chai.as.promises).